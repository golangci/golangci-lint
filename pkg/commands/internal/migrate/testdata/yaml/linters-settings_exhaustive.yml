issues:
  # Only to not generate unrelated elements inside golden.
  exclude-use-default: false
  # Only to not generate unrelated elements inside golden.
  exclude-generated: strict
  # Only to not generate unrelated elements inside golden.
  exclude-dirs-use-default: false

linters-settings:
  exhaustive:
    check:
      - switch
      - map
    check-generated: true
    default-signifies-exhaustive: true
    ignore-enum-members: "Example.+"
    ignore-enum-types: "Example.+"
    package-scope-only: true
    explicit-exhaustive-switch: true
    explicit-exhaustive-map: true
    default-case-required: true
