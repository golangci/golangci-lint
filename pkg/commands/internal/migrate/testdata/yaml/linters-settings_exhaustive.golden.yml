version: "2"
linters:
  settings:
    exhaustive:
      check:
        - switch
        - map
      default-signifies-exhaustive: true
      ignore-enum-members: Example.+
      ignore-enum-types: Example.+
      package-scope-only: true
      explicit-exhaustive-map: true
      explicit-exhaustive-switch: true
      default-case-required: true
